client-js/test/spec/tsTestSpec.ts

```diff
@@ -0,0 +1,42 @@
+import {Agent} from '../test/helloWorld';
+import {Observable} from '../app/scripts/observable';
+
+
+describe("Hello world", function() {
+  var agent;
+
+  beforeEach(function() {
+    agent = new Agent();
+  });
+
+  it("says hello", function() {
+    expect(agent.helloWorld()).toEqual("Hello world!");
+  });
+});
+
+/*
+* Test to create new observable and add callback
+* This is mostly to see if the plumbing for the
+* tests interfaces with the scripts in /app
+*/
+
+describe("observable", function() {
+  var observable;
+
+  beforeEach(function() {
+    observable = new Observable<any>();
+
+  });
+
+  function testFunc(message: string) {
+    console.log("testFunc called");
+    return message;
+  }
+
+
+  it("Add Callback", function() {
+    console.log("In callback test");
+    expect(observable.add(testFunc("something"))).toEqual("something");
```

Comment:
So, I'm not sure what's going on, but the compiler is doing some type checking, if I change: 
```
observable.add(testFunc("something"))
````
to 
```
observable.add(testFunc(4))
```
I get the compiler error: ```test/spec/tsTestSpec.ts(39,36): error TS2345: Argument of type 'number' is not assignable to parameter of type 'string'.```

Now what I don't know if it's doing the type checking properly for the imports? Maybe something is mis-configured.